### DISCUSSION TITLE
Client Side caching

### DISCUSSION THREAD

**Discussion by arielzach** [redis/rueidis#367] [General]
Hello, I have some questions:

- Does Client Side Caching work in cluster mode?
- Is DisableCache true by default?
- Does using side caching have any CPU and memory usage overhead?

**Comment by rueian** [comment]
1. Yes, Client Side Caching works in cluster mode
2. DisableCache is false by default. However, you still need to use `DoCache` explicitly for Client Side Caching.
3. Yes, it will put additional CPU and memory usage on both sides of the client and server.
